src/LibraryManager.Vsix/Commands/CommandTable/VSCommandTable.en.vsct

```diff
@@ -12,7 +12,9 @@
 
         <Buttons>
             <!-- Folder -->
-            <Button guid="guidLibraryManagerPackageCmdSet" id="InstallPackage" priority="0x0100" type="Button">
```

Comment:
Looks like I didn't fix this file during the loc check-in (fixed loc-ed VSCT files, but not the English one). We need to make sure this makes it into Preview 3.